low-level/solution/lr2/server/manifest.py

```diff
@@ -296,12 +298,12 @@ def get_service_info(self, services):
                 uid, _, health_description, _, specifics = response
                 service_info = {
                     "uid": uid,
-                    "type": "NA",
+                    "type": "Not Applicable",
                     "description": health_description,
                     "product": specifics[0].pop("service_name"),
-                    "manufacturer": "NA",
-                    "serial_number": "NA",
-                    "part_number": "NA",
+                    "manufacturer": "Not Applicable",
```

Comment:
Shouldn't be 'Not Available'?

Reply:
For services, we don't have 'manufacturer'

Reply:
then still do we have to keep those fields? manufacturer, serial_number, part_number??

Reply:
Okay